.gitlint

```diff
@@ -0,0 +1,13 @@
+[general]
+ignore=title-trailing-punctuation, body-min-length, body-is-missing, title-imperative-mood
```

Comment:
Did you mean to exclude `title-imperative-mood`? Is this is from zulip/zulip? It seems strange to exclude a rule that's added in the extra rules.

Reply:
Yeah, the code is mostly from Zulip. If you have a preference please mention the repository. I will figure out the changes required.

Reply:
I am taking a look and trying to make the required changes. Thanks for the review.

Reply:
This is wrong in `zulip/zulip`; see zulip/zulip#17393.  Once that merges, please bring those changes into this PR.